nix/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
developer/maintainer oriented docs pointing at Niv docs and/or giving brief summary of how to update dependencies with Niv would probably be helpful

Reply:
Done.